authentication/views_test.py

```diff
@@ -386,7 +410,7 @@ def login_password_user_inactive(self, auth_state, verify_exports):
     #         "authentication.pipeline.compliance.api.verify_user_with_exports",
     #         side_effect=Exception("register_details_export_temporary_error"),
     #     ):
-    #         assert_api_call(
+    #         assert_api_call_json(
```

Comment:
I know the commented test is not part of this PR, but if possible for you, I think we should remove it or add a proper TODO/Comment on the test if we really want to keep it like this.